.github/actions/run-integration-tests/action.yml

```diff
@@ -19,15 +19,8 @@ runs:
   using: "composite"
   steps:
     - name: Set IMAGE_TAG
```

Comment:
I think it is now more like a "Get" or "Compute" instead of "Set" because we are no anymore setting it in the environment variables but just computing/getting the value.